apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -151,9 +157,13 @@ function getParamsForAssessment(
             maxAutoPoints: question.maxAutoPoints ?? null,
             autoPoints: question.autoPoints ?? null,
             manualPoints: question.manualPoints ?? null,
-            forceMaxPoints: question.forceMaxPoints || false,
-            triesPerVariant: question.triesPerVariant || 1,
-            advanceScorePerc: question.advanceScorePerc,
+            forceMaxPoints: question.forceMaxPoints ?? false,
+            triesPerVariant: question.triesPerVariant ?? 1,
+            advanceScorePerc:
+              question.advanceScorePerc ??
+              zone.advanceScorePerc ??
+              assessment.advanceScorePerc ??
+              0,
```

Comment:
I don't think we want this change. We always want the raw value here. This coalescing is being done later when we compute `effective_advance_score_perc`.

The fact that this didn't cause a test to fail is concerning. IMO we should add a test that fails with this new behavior but passes with the old one.

Reply:
I added such a test in #11794. The test then started failing on this branch, I removed these changes in 6925bc4 to get the tests passing again.